cmd/ig/utils/flags.go

```diff
@@ -88,6 +95,9 @@ func AddCommonFlags(command *cobra.Command, commonFlags *CommonFlags) {
 			commonFlags.RuntimeConfigs = append(commonFlags.RuntimeConfigs, &containerutilsTypes.RuntimeConfig{
 				Name:       runtimeName,
 				SocketPath: socketPath,
+				Extra: &containerutilsTypes.ExtraConfig{
+					Namespace: namespace,
+				},
 			})
```

Comment:
I wouldn't even pass the ExtraConfig struct if the namespace is nil. Something like:

```go
r := &containerutilsTypes.RuntimeConfig{
	Name:       runtimeName,
	SocketPath: socketPath,
}

if namespace != "" {
	r.Extra = &containerutilsTypes.ExtraConfig{
		Namespace: namespace,
	}
}

commonFlags.RuntimeConfigs = append(commonFlags.RuntimeConfigs, r)
```

Reply:
Nice. It should be this way, makes the right use of pointer for `ExtraConfig`.